url/Cargo.toml

```diff
@@ -25,11 +25,14 @@ bencher = "0.1"
 
 [dependencies]
 form_urlencoded = { version = "1.0.0", path = "../form_urlencoded" }
-idna = { version = "0.2.0", path = "../idna" }
+idna = { version = "0.2.0", path = "../idna", optional = true }
 matches = "0.1"
 percent-encoding = { version = "2.1.0", path = "../percent_encoding" }
 serde = {version = "1.0", optional = true, features = ["derive"]}
 
+[features]
+default = ["idna"]
```

Comment:
I wonder if we should reverse this and name the feature "disable-idna" or "unsafe-disable-idna" ?